manifest.json

```
@@ -0,0 +1,10 @@
+{
+  "id": "obsidian-feeds",
+  "name": "Obsidian Feeds",
+  "version": "0.0.1",
+  "minAppVersion": "1.2.0",
+  "description": "Magic feeds dataview query for obsidian",
```

[🔧](https://github.com/erikthedeveloper/code-review-emoji-guide#:~:text=:wrench:)
```suggestion
  "description": "Obsidian plugin to add magic feed views using Dataview",
```